packages/cosmic-swingset/lib/ag-solo/start.js

```diff
@@ -123,23 +123,42 @@ async function buildSwingset(
     }
   }
 
+  // This synchronises the processKernel calls between
+  // deliverInboundToMbx and deliverInboundCommand.
+  let lastProcessKernelP = Promise.resolve();
```

Comment:
I can't visualize this control flow in my head.. does it preserve the ordering of inbound calls when there are two or more queued up?

Reply:
It was half-baked and wishful.  I've reverted to the prior implementation and TODO comment.

Reply:
Nope, I changed my mind and went for the fully-baked version.